semgrep-core/src/analyzing/Dataflow_tainting.mli

```diff
@@ -6,6 +6,10 @@ type fun_env = (Dataflow.var, unit) Hashtbl.t
   * Note that here [Dataflow.var] is a string of the form "<source name>:<sid>". *)
 
 type config = {
+  opaque_is_safe : bool;
+      (** If [opaque_is_safe] taint tracking will assume that third functions, that
```

Comment:
third-party

Reply:
but really it's not even third-party, it's just every functions.

Reply:
Yeah I am not entirely happy with any term. I say "opaque" in many places because there are functions like `Concat` that we understand, whereas other functions are "opaque" to Semgrep and those are the ones affected by this option. But then I use "third(-party)" in others because I'm not sure whether users will understand what is "opaque" here.

Reply:
What about "other opaque functions", despite being a bit long. Config option could be `other_opaque_funs_are_safe`?